google/resource_dataproc_cluster.go

```diff
@@ -88,6 +88,16 @@ func resourceDataprocCluster() *schema.Resource {
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 
+						"delete_autogen_bucket": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  false,
+							Removed: "autogenerated buckets are shared by all clusters in the same region, " +
```

Comment:
This message can probably be updated? We don't need to explain why this is a bad idea anymore, we just need to state you can't.